flavors/standard-EXASOL-7.1.0/flavor_base/base_test_deps/packages/apt_get_packages

```diff
@@ -1,7 +1,7 @@
-gdb|8.1.1-0ubuntu1
-valgrind|1:3.13.0-2ubuntu2.3
-gdbserver|8.1.1-0ubuntu1
-binutils|2.30-21ubuntu1~18.04.9
-patchelf|0.9-1
+gdb|9.2-0ubuntu1~20.04.1                                                                                                 
+valgrind|1:3.15.0-1ubuntu9.1                                                                                        
+gdbserver|9.2-0ubuntu1~20.04.1                                                                                           
+binutils|2.34-6ubuntu1.6                                                                                         
+patchelf|0.10-2build1                                                                                                             
```

Comment:
```suggestion
gdb|9.2-0ubuntu1~20.04.1
valgrind|1:3.15.0-1ubuntu9.1
gdbserver|9.2-0ubuntu1~20.04.1
binutils|2.34-6ubuntu1.6
patchelf|0.10-2build1
```

trailing whitespace

Reply:
Ah, that comes from copying out of tmux, I hoped I got all